src/ergw_aaa_nasreq.erl

```diff
@@ -394,6 +394,10 @@ to_session(_, Key = '3GPP-IPv6-DNS-Servers', [Value], {Session, Events}) ->
     {Session#{Key => ergw_aaa_3gpp_dict:decode(Key, Value)}, Events};
 to_session(_, 'TP-NAT-Pool-Id', [Id], {Session, Events}) ->
     {Session#{'NAT-Pool-Id' => Id}, Events};
+to_session(_, 'Framed-Pool', [Id], {Session, Events}) ->
+    {Session#{'Framed-Pool' => Id}, Events};
+to_session(_, 'Framed_IPv6_Pool', [Id], {Session, Events}) ->
+    {Session#{'Framed_IPv6_Pool' => Id}, Events};
```

Comment:
This looks wrong. Are you sure it should use underscores?

Reply:
thanks, fixed